examples/fluids/problems/freestream_bc.c

```diff
@@ -104,8 +97,7 @@ PetscErrorCode FreestreamBCSetup(ProblemData *problem, DM dm, void *ctx, Newtoni
   CeedQFunctionContextCreate(user->ceed, &freestream_context);
   CeedQFunctionContextSetData(freestream_context, CEED_MEM_HOST, CEED_USE_POINTER, sizeof(*freestream_ctx), freestream_ctx);
   CeedQFunctionContextSetDataDestroy(freestream_context, CEED_MEM_HOST, FreeContextPetsc);
-  problem->apply_freestream.qfunction_context          = freestream_context;
-  problem->apply_freestream_jacobian.qfunction_context = freestream_context;
```

Comment:
@jrwrigh I think this is your double free bug when multiple faces have freestream boundary conditions.